### PR TITLE
update argocd version form 2.11.7 to 2.11.9

### PR DESCRIPTION
**Pull request by anandrkskd**
**What type of PR is this?**

 /kind chore

**What does this PR do / why we need it**:
This PR updates the argocd version for release-0.11 from version 2.11.7 to 2.11.9
**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: config/crd/bases/argoproj.io_applications.yaml [hunk 1]
```diff
@@ -12,519 +12,176 @@ spec:
     listKind: ApplicationList
     plural: applications
     shortNames:
-    - app
-    - apps
+      - app
```
Comment (anandf) [inline config/crd/bases/argoproj.io_applications.yaml:15]: These are formatting changes. May be done via auto formatting. These may not be required.
https://github.com/argoproj/argo-cd/blob/v2.11.9/manifests/crds/application-crd.yaml

File: config/crd/bases/argoproj.io_applications.yaml [hunk 2]
```diff
@@ -12,519 +12,176 @@ spec:
...
-                      operation
-                    type: string
-                type: object
-              retry:
-                description: Retry controls the strategy to apply if a sync fails
-                properties:
-                  backoff:
-                    description: Backoff controls how to backoff on subsequent retries
-                      of failed syncs
+    - additionalPrinterColumns:
```
Comment (anandf) [inline config/crd/bases/argoproj.io_applications.yaml:20]: Same as above. Remove formatting changes.

File: config/crd/bases/argoproj.io_applications.yaml [hunk 3]
```diff
@@ -28,10 +28,6 @@ spec:
       name: Revision
       priority: 10
       type: string
-    - jsonPath: .spec.project
-      name: Project
-      priority: 10
-      type: string
```
Comment (svghadi) [inline config/crd/bases/argoproj.io_applications.yaml:34]: Can you run `make generate manifests bundle` so that change is reflected into manifest files?

**Review by svghadi — APPROVED** [review]
LGTM. Thanks

**Pull request merged**